libhb/muxavformat.c

```diff
@@ -1184,7 +1195,9 @@ static int avformatMux(hb_mux_object_t *m, hb_mux_data_t *track, hb_buffer_t *bu
         return 0;
     }
 
-    if (track->type == MUX_TYPE_VIDEO && (job->mux & HB_MUX_MASK_MKV) &&
+    // @TODO: Find out if libav's webm muxing is subject to this.
```

Comment:
This is still a problem, but it's even worse now.  Audio also needs to be looked at because we can get negative audio PTS due to audio pre-roll.  I'll fix this in another PR.  The init_delay for every stream needs to be looked at and an offset computed that gets applied to pts and chapter times.